Resources/Prototypes/Catalog/VendingMachines/Inventories/medical.yml

```diff
@@ -9,4 +9,5 @@
     Syringe: 5
     ClothingEyesHudMedical: 2
     ClothingEyesEyepatchHudMedical: 2
-
+  contrabandInventory:
+    Defibrillator: 1
```

Comment:
Its probably not a good idea to have this many defibs, and having it in here makes it hard to balance for medbay.

Reply:
Defib is literally 3 steel, its way easier to get it printed most of the time, so it would be more of an emergency defib.

Reply:
thebadman4662 is right, the idea of this is that it's an emergency defibrillator that a Greytider/Salvager/Engineer can pop out of a NanoMed Plus during an emergency, where there's either power or they've gone to the lengths of finding a generator to power it. I'm actually still surprised with how cheap defibs are to make.

Defibs are cheap, but NanoMed Plus machines are common, and honestly the medium power cell inside is probably the most valuable part of it. I don't see people just going into medbay, hacking every NanoMed Plus and kicking out all of the defibs. However, in such a world I'd be fine adding singles of medicated sutures and regenerative mesh (1x stack of 1) as a "tease" item and as an accessible instructional tool for teaching medical interns. Otherwise, I need more ideas on what should go in here.

To note, my proposal has the WallMed/NanoMed machines getting 2x small power cell, specifically to charge a defib for one jolt in an emergency.

Reply:
Replaced NanoMed defibrillator for 1-of singles of medicated suture and regenerative mesh.